tools/state-viewer/src/cli.rs

```diff
@@ -651,6 +652,43 @@ impl clap::ValueEnum for ViewTrieFormat {
     }
 }
 
+#[derive(Clone, IntoStaticStr)]
+pub enum StateRecordType {
+    Account,
+    Data,
+    Contract,
+    AccessKey,
+    PostponedReceipt,
+    ReceivedData,
+    DelayedReceipt,
+}
+
+impl clap::ValueEnum for StateRecordType {
+    fn value_variants<'a>() -> &'a [Self] {
+        &[
+            Self::Account,
+            Self::Data,
+            Self::Contract,
+            Self::AccessKey,
+            Self::PostponedReceipt,
+            Self::ReceivedData,
+            Self::DelayedReceipt,
+        ]
+    }
```

Comment:
note:
I thought about reusing an enum already defined elsewhere, but there are problems with that, e.g.
I cannot define `clap::ValueEnum` trait for an enum from external crate.